js/Features.js

```diff
@@ -52,19 +42,19 @@ var Features = function (config) {
 
     self._features = features;
     self._featureMask = 0;
-}
+};
```

Comment:
Ok, I can change it back. It was actually WebStorm that complained that the statement was unterminated.